integration-tests/src/test/scala/za/co/absa/spline/ViewAttributeLineageSpec.scala

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package za.co.absa.spline
+
+import org.scalatest.OneInstancePerTest
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+import za.co.absa.spline.test.LineageWalker
+import za.co.absa.spline.test.ProducerModelImplicits._
+import za.co.absa.spline.test.SplineMatchers._
+import za.co.absa.spline.test.fixture.spline.SplineFixture
+import za.co.absa.spline.test.fixture.{SparkDatabaseFixture, SparkFixture}
+
+class ViewAttributeLineageSpec
+  extends AsyncFlatSpec
+    with OneInstancePerTest
+    with Matchers
+    with SparkFixture
+    with SparkDatabaseFixture
+    with SplineFixture {
+
+  "persistent view" should "not break attribute dependency" in
```

Comment:
Think about naming tests after what they _should_ do instead of what they shouldn't.
E.g.
```scala
it should "support persistent views" // or
it should "trace attribute dependencies through the persistent views"
```